scripts/agent-trader/agent_trader.py

```
@@ -154,18 +154,36 @@ def _fetch_market(market_id):
+    the summary/email/metrics disclose those without touching the resolved-only stats.
+    """
     bets = load_bets()
     changed = False
     for b in bets:
         if b["status"] != "open":
             continue
-        m = _fetch_market(b["market_id"])
-        if not m or not m.get("closed"):
+        try:
+            m = _fetch_market(b["market_id"])
+        except Exception as e:                      # transient network/proxy failure
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)